code/game/machinery/cryopod.dm

```diff
@@ -335,7 +339,7 @@
 
 		var/preserve = null
 		for(var/T in preserve_items)
-			if(istype(W,T))
+			if(istype(W,T) && !(W in do_not_preserve_items))
```

Comment:
I guess, but that's (as you said) snowflake-y, as the code doesn't have other special checks for items.